package-lock.json

```diff
@@ -0,0 +1,1587 @@
+{
```

Comment:
Do we want to have lockfiles checked into Git for MentorQ? If we do then we should have one for every `package.json`, including `api/package-lock.json` and `client/package-lock.json`. If we don't then we should ignore `package-lock.json`. Not arguing for or against, but some actions seem to be needed.

Reply:
Hmmm because we're using Docker, usually the package-lock.json wouldn't be generated until npm install is ran, and it wouldn't be part of the repo. This one is probably due to running some local npm install. Maybe just remove this from source control?

Reply:
@shannonphu How would I do that?

Reply:
@dustinnewman98 You would add a `package-lock.json` line to the `.gitignore` file.